docs/user/release_notes.rst

```diff
@@ -2,6 +2,26 @@
 Release notes
 =============
 
+v4.0.1
+------
+
+This is a bugfix release.
+
+
+Bug fixes
+^^^^^^^^^
+
+* Fix zchunk updateinfo getting injected as updateinfo_zck
```

Comment:
Let's add a link to this pull request in case anyone wants to reference it later for more details.

```suggestion
* Fix zchunk updateinfo getting injected as updateinfo_zck (:issue:`3261`).
```